src/Subscriptions/Storage/RedisStorageManager.php

```diff
@@ -38,7 +38,14 @@ public function __construct(ConfigRepository $config, RedisFactory $redis)
         $this->connection = $redis->connection(
             $config->get('lighthouse.subscriptions.broadcasters.echo.connection') ?? 'default'
         );
-        $this->ttl = $config->get('lighthouse.subscriptions.storage_ttl');
+        $ttl = $config->get('lighthouse.subscriptions.storage_ttl');
+        if (is_int($ttl)) {
+            $this->ttl = $ttl;
+        } elseif (is_string($ttl) && is_numeric($ttl)) {
+            $this->ttl = (int) $ttl;
+        } else {
+            throw new \ErrorException('ttl is not an integer');
```

Comment:
```suggestion
            $notIntOrNumericString = Utils::printSafe($ttl);
            throw new \Exception("Expected config option lighthouse.subscriptions.storage_ttl to be an int or a numeric string, got: {$notIntOrNumericString}.");
```

`GraphQL\Utils\Utils`